src/pages/guides/rpcs-and-das.md

```diff
@@ -0,0 +1,74 @@
+---
+title: RPCs and DAS
```

Comment:
Page title is RPCs and DAS, but DAS is not mentioned in the page?

Reply:
... not yet it's not. We can add an overview of DAS quickly if need be. I just want to get the main part out.